### DISCUSSION TITLE
How to align <Select> placeholder to center

### DISCUSSION THREAD

**Discussion by ryanalcova** [chakra-ui/chakra-ui#4242] [Q&A]
Hello guys,

Is there anyway that I can apply the textAlign: 'center' to the `<Select>` placeholder?

![Screen Shot 2021-06-16 at 6 57 04 PM](https://user-images.githubusercontent.com/71019733/122305930-9f3f1700-ced5-11eb-84e6-17433e6fd9cb.png)

**Comment by anubra266** [comment]
![image](https://user-images.githubusercontent.com/30869823/122306194-ee9a3d00-ceff-11eb-826a-4b26eb9caa9e.png)

**Reply by ryanalcova** [reply]
Hmm.... it is not working here. Not even on the ChakraUI website.

**Chakra Website:**
![Screen Shot 2021-06-16 at 7 25 44 PM](https://user-images.githubusercontent.com/71019733/122307713-e1b62300-ced8-11eb-8586-489b6d0b9e97.png)

**In my project:"**

![Screen Shot 2021-06-16 at 7 24 36 PM](https://user-images.githubusercontent.com/71019733/122307753-f1ce0280-ced8-11eb-8281-6b8334583f56.png)
![Screen Shot 2021-06-16 at 7 24 46 PM](https://user-images.githubusercontent.com/71019733/122307777-f85c7a00-ced8-11eb-98ba-5af9066bc710.png)

**Reply by anubra266** [reply]
What browser are you using?

**Reply by ryanalcova** [reply]
Chrome on Mac OS, Version 91.0.4472.106 (Official Build) (x86_64).
